phalcon/forms/form.zep

```diff
@@ -290,17 +290,19 @@ class Form extends Injectable implements \Countable, \Iterator
 			}
 		}
 
-		let notFailed = true,
-			messages = [];
+		let validationStatus = true;
+
+        let validation = this->getValidation();
+
+        if !(validation instanceof Validation) {
+            // Create an implicit validation
+            validation = new Validation();
```

Comment:
`let` statement is used to mutate variables, properties and arrays. Variables are by default immutable and this instruction makes them mutable:
```zephir
let validation = new Validation();
```